runner.sh

```diff
@@ -1,11 +1,33 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
-BIN=$1
-shift 1
+# script to help run the binary with specific jvm and jvm options
+# Defaults to openj9. Turn on hotspot by setting env JVM_HOTSPOT
 
-echo $@
+JVM=$1
+BIN=$2
```

Comment:
this would be less confusing as `JAVA_BIN` and `JAR`, respectively (JVM is also a BIN and not used to seeing "java" bin referred to as "JVM")